image/image.go

```diff
@@ -166,15 +231,54 @@ func createRuntimeBundle(w walker, dest, refName, rootfs string) error {
 		return err
 	}
 
-	m, err := findManifest(w, ref)
-	if err != nil {
-		return err
+	if ref.MediaType == validRefMediaTypes[0] {
+		m, err := findManifest(w, ref)
+		if err != nil {
+			return err
+		}
+
+		if err := m.validate(w); err != nil {
+			return err
+		}
+
+		return createRuntimebundle(w, m, dest, rootfs)
 	}
 
-	if err = m.validate(w); err != nil {
-		return err
+	if ref.MediaType == validRefMediaTypes[1] {
+		ml, err := findManifestList(w, ref)
+		if err != nil {
+			return err
+		}
+
+		if err := ml.validate(w); err != nil {
+			return nil
+		}
+
+		if len(ml.Manifests) == 0 {
+			fmt.Println("warning: no manifests found")
+			return nil
+		}
+
+		for _, manifest := range ml.Manifests {
+			m, err := findManifest(w, &(manifest.Descriptor))
+			if err != nil {
+				return err
+			}
+
+			if err := m.validate(w); err != nil {
+				return err
+			}
+
+			if err := createRuntimebundle(w, m, dest, rootfs); err != nil {
+				return err
```

Comment:
Is not able to contain all the manifest are listed, let the user choose which one to use.